docs/src/develop/plugins/server_plugin_api.md

```diff
@@ -529,7 +529,7 @@ app.resourcesApi.setResource(
   'waypoints',
   'ac3a3b2d-07e8-4f25-92bc-98e7c92f7f1a',
   {
-    "position": {"longitude": 138.5, "latitude": -38.6}, 
+    "name": "My Waypoint",
     "feature": {
```

Comment:
Did you intend this to be included?

Reply:
It is a valid correction (position is not part of the V2 waypoint definition), but should have been a separate commit.